gtsfm/data_association/point3d_initializer.py

```diff
@@ -85,14 +120,15 @@ def execute_ransac_variant(self, track_2d: SfmTrack2d) -> np.ndarray:
             best_inliers: boolean array of length N. Indices of measurements
                are set to true if they correspond to the best RANSAC hypothesis
         """
-        if self.num_ransac_hypotheses is None:
-            raise ValueError("RANSAC triangulation requested but number of hypothesis is None.")
+        # if self.num_ransac_hypotheses is None:
+        #     raise ValueError("RANSAC triangulation requested but number of hypothesis is None.")
 
         # Generate all possible matches
         measurement_pairs = generate_measurement_pairs(track_2d)
 
         # limit the number of samples to the number of available pairs
-        num_hypotheses = min(self.num_ransac_hypotheses, len(measurement_pairs))
+        num_hypotheses = min(self.options.num_ransac_hypotheses(), len(measurement_pairs))
+        logger.debug(num_hypotheses)
```

Comment:
Would be helpful to make this log message more descriptive, e.g. 
```python
logger.debug("Num RANSAC hypotheses for triangulation: %d", num_hypotheses)
```

Reply:
This was just for debugging purposes; I will remove it.

Reply:
removed